src/app/customer/Customer.tsx

```diff
@@ -49,52 +51,90 @@ export interface WithCheckoutCustomerProps {
     createAccountError?: Error;
     signInError?: Error;
     clearError(error: Error): Promise<CheckoutSelectors>;
-    continueAsGuest(credentials: GuestCredentials): Promise<CheckoutSelectors>;
+    continueAsGuest(credentials: GuestCredentials, options: CustomerRequestOptions): Promise<CheckoutSelectors>;
     deinitializeCustomer(options: CustomerRequestOptions): Promise<CheckoutSelectors>;
     initializeCustomer(options: CustomerInitializeOptions): Promise<CheckoutSelectors>;
+    loadPaymentMethods(): Promise<CheckoutSelectors>;
     sendLoginEmail(params: { email: string }): Promise<CheckoutSelectors>;
-    signIn(credentials: CustomerCredentials): Promise<CheckoutSelectors>;
-    createAccount(values: CustomerAccountRequestBody): Promise<CheckoutSelectors>;
+    signIn(credentials: CustomerCredentials, options: CustomerRequestOptions): Promise<CheckoutSelectors>;
+    createAccount(values: CustomerAccountRequestBody, options: CustomerRequestOptions): Promise<CheckoutSelectors>;
 }
 
 export interface CustomerState {
+    isReady: boolean;
     isEmailLoginFormOpen: boolean;
     hasRequestedLoginEmail: boolean;
 }
 
 class Customer extends Component<CustomerProps & WithCheckoutCustomerProps, CustomerState> {
     state: CustomerState = {
+        isReady: false,
         isEmailLoginFormOpen: false,
         hasRequestedLoginEmail: false,
     };
 
     private draftEmail?: string;
+    private providerIdWithCustomContinueFlow?: string;
 
-    componentDidMount(): void {
+    async componentDidMount(): Promise<void> {
         const {
-            onReady = noop,
             email,
+            loadPaymentMethods,
+            initializeCustomer,
+            onReady = noop,
+            onUnhandledError = noop,
         } = this.props;
 
         this.draftEmail = email;
 
+        try {
+            const { data } = await loadPaymentMethods();
```

Comment:
I'm not sure if we want to do this any differently than how we handle Masterpass and GooglePay. These remote checkout providers are listed under `config.checkoutSettings.remoteCheckoutProviders` so we don't need to make another call to load all the payment methods enabled for the store.

Reply:
I didn't know that. I will check this out

Reply:
Maybe we need to add new `config.checkoutSettings.customContinueFlowPovider`?

I need to check, if provider contains `customerContinueFlowEnabled` flag and that it is on.

Reply:
Or I can filter all `remoteCheckoutProviders` (if we will ad bolt there) on `customerContinueFlowEnabled` flag that will not be used in

Reply:
@davidchin @Sireks here is PR ([41748](https://github.com/bigcommerce/bigcommerce/pull/41748)) for this changes.